### ISSUE TITLE
Show package version in use next to required packages

### ISSUE THREAD

**Issue by achristodoulou** [composer/satis#245]
Hi,

It would be very helpful if you could show the package version currently in use next to the required by packages.

For example:

I have the following 2 repos in my satis config. Application achristodoulou/my-application uses achristodoulou/my-package and having next to the package in required by the version as well it would be very helpful to identify which project is outdated and composer update is required.

```
--------------------------
achristodoulou/my-package

Releases	dev-master, 4.2.1, v4.2.0, v4.1.0, v4.0.1, v4.0.0, v3.0.4, v3.0.3, v3.0.2, v3.0.1, v3.0.0, v2.0.1, v2.0.0, v1.0.1, v1.0.0
Required by: achristodoulou/my-application(v4.2.0) <---- show which version it uses
--------------------------
achristodoulou/my-application

Releases	dev-master
Required by: none
--------------------------
```

Thanks

**Comment by alcohol** [comment]
`composer show -i`

**Comment by achristodoulou** [comment]
Thanks but I was not referring to command line, but to display this information in Satis static web pages instead.

**Comment by alcohol** [comment]
Yeah sorry, I don't see any value there. You're welcome to try and write it yourself, submit a PR :-)
